third_party/plugin/src/test/BUILD

```diff
@@ -12,10 +12,10 @@ scala_junit_test(
                 "//external:io_bazel_rules_scala/dependency/scala/scala_compiler",
                 "//external:io_bazel_rules_scala/dependency/scala/scala_library",
                 "@com_google_guava_guava_21_0_with_file//jar",
-                "@org_apache_commons_commons_lang_3_5_without_file//jar"
+                "@org_apache_commons_commons_lang_3_5_without_file//:linkable_org_apache_commons_commons_lang_3_5_without_file"
```

Comment:
@ittaiz please note that for `linkable` imports the label cannot use `jar` alias